.github/workflows/cloud_csharp_client_test.yaml

```diff
@@ -0,0 +1,100 @@
+name: Hazelcast cloud test with csharp client
+
+on:
+  workflow_dispatch:
+    inputs:
+      organization_name:
+        description: Default is hazelcast, but if you would like to run the workflow with your forked repo, set your github username
+        required: true
+        default: hazelcast
+      branch_name:
+        description: Name of the branch to test client from
+        required: true
+        default: master
+      base_url:
+        description: Base url of the cloud env. I.e https://uat.hazelcast.cloud
+        required: true
+        default: https://uat.hazelcast.cloud
+      hzVersion:
+        description: Version of hazelcast (For cloud standard cluster, it has different versions, i.e 5.0.2-2)
+        required: true
+
+jobs:
+  test_standard_cluster:
+    runs-on: ubuntu-latest
+    name: Hazelcast cloud standard cluster tests with csharp
+    strategy: 
+      matrix: 
+        fwks: [netcoreapp3.1,net5.0,net6.0]
+    steps:
+      - name: Checkout to scripts
+        uses: actions/checkout@v2
+        
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.10
+          
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+          
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 6.0
```

Comment:
I see that we don't set up other dotnet versions (netcoreapp3.1 and net5.0). Will they be installed automatically when we install just 6.0?

Reply:
Yes, 6.0 is backward compatible. But I doubt now in point of view QA/Testing.

Reply:
I meant, you just install 6.0, but run tests with different .net versions (.net core 3.1 and 5.0). How can this work?